### PR TITLE
The check for ignoring should use the resource block

### LINKED ISSUE #781: tfsec `v0.40.01` ignore rules inconsistent - raises error code on `AWS093` and `AWS032`
**Describe the bug**
Our CI builds started failing yesterday with the release of tfsec v0.40.0. Where v0.39.42 ran fine previously, the recently released v0.40.0 and v0.40.1 are causing us issues. Specifically, we were ignoring `AWS093` and `AWS032` but started seeing their respective error messages pop up when running tfsec in our CI builds, which leads to failures. I installed the latest version via brew (`v0.40.1`) and verified locally the same error is occurring. 

Further testing indicates that the rules are successfully ignored when passed through the command line using the `-e` flag but if they are added into the terraform file via `#tfsec:ignore:AWS093`, they aren't being ignored.

**To Reproduce**
1. Install tfsec `v0.40.01` or up
2. Run `tfsec` against a terraform file which contains `#tfsec:ignore:AWS093` -> see an error
3. Try running `tfsec -e AWS093` against that same file -> see no error as expected

**Expected behavior**
tfsec should ignore the rules as defined using the `#tfsec:ignore:XXXXXX` format

**Screenshots**

**Desktop (please complete the following information):**
 - OS: Linux docker images, Mac OS

**Additional context**

Below is a snippet of the output with our code removed:

```sh
Check 1

  [AWS093][INFO] Resource 'aws_ecr_repository.bert' does not have CMK encryption configured
  /Users/johannes.giorgis/src/professional/ernie/services/bert/main.tf:27-35


<...relevant code snippet skipped...>

  Impact:     Using AWS managed keys does not allow for fine grained control
  Resolution: Use customer managed keys


Check 2

  [AWS093][INFO] Resource 'aws_ecr_repository.bert' should have the encryption type set to KMS
  :0
```

### PR DESCRIPTION
**Pull request by owenrumney**
- take the range details from the resource block rather than the result
block

- add a check for someone adding the tfsec ignore in about the result
block

Resolves #781

### CODE REVIEW

File: internal/app/tfsec/scanner/scanner.go [hunk 1]
```diff
@@ -111,6 +111,14 @@ func (scanner *Scanner) checkRangeIgnored(id string, r block.Range, b block.Rang
 	}
 
 	// check the line above the block
+	if r.StartLine-1 > 0 {
```
Comment (liamg) [inline internal/app/tfsec/scanner/scanner.go:114]: It looks like this block already exists below 🤔

Comment (liamg) [reply internal/app/tfsec/scanner/scanner.go:114]: Nope, I can't read

**Review by liamg — DISMISSED** [review]

**Comment by owenrumney** [comment]
Yup @liamg  - that seems to have been unnecessary after all. The key part is the range filename

**Review by liamg — APPROVED** [review]
Ah I see!

**Pull request merged**